docs/sources/userguide/plugins.md

```diff
@@ -0,0 +1,157 @@
+Docker Plugins MVP
+==================
+
+Plugins provide a mechanism for hooking into the Docker engine to extend its behavior.
+
+HTTP over a local UNIX socket is used to communicate between Docker and its plugins.
+
+Plugins are distributed as containers.
+
+This is targeted at Docker 1.7.
+
+# Usage
+
+Plugins can be loaded using a special `docker plugin load` command, as follows:
+
+```
+$ docker plugin load -e $ARGS clusterhq/flocker-plugin $@
+[... fetching...]
+Plugin flocker:v0.1 loaded and registered with extension points:
+* volumes
+Plugin flocker activated.
+```
+
+This is really just syntactic sugar for the following:
+
+```
+$ docker run -d -e $ARGS \
+	-v /var/run/docker-plugins/flocker.sock:/var/run/plugin.sock \
```

Comment:
why not use /var/lib/docker ?

On Friday, March 6, 2015, Ben Firshman <notifications@github.com> wrote:

> In docs/sources/userguide/plugins.md
> <https://github.com/cpuguy83/docker/pull/3#discussion_r25964675>:
>
> > +
> > +Plugins can be loaded using a special `docker plugin load` command, as follows:
> > +
> > +```
> > +$ docker plugin load -e $ARGS clusterhq/flocker-plugin $@
> > +[... fetching...]
> > +Plugin flocker:v0.1 loaded and registered with extension points:
> > +* volumes
> > +Plugin flocker activated.
> > +```
> > +
> > +This is really just syntactic sugar for the following:
> > +
> > +```
> > +$ docker run -d -e $ARGS \
> > +	-v /var/run/docker-plugins/flocker.sock:/var/run/plugin.sock \
>
> Perhaps /var/run/docker/ should be a thing? /var/run/docker/api.sock,
> /var/run/docker/plugins/, etc
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cpuguy83/docker/pull/3/files#r25964675>.
>